mlir/test/Dialect/Vector/invalid.mlir

```diff
@@ -271,6 +304,38 @@ func.func @insert_0d(%a: f32, %b: vector<f32>) {
   %1 = vector.insert %a, %b[0] : f32 into vector<f32>
 }
 
+// -----
+func.func @extract_vector_mixed_index_types(%arg0 : f32, %arg1 : vector<8x16xf32>,
```

Comment:
```suggestion
func.func @insert_vector_mixed_index_types(%arg0 : f32, %arg1 : vector<8x16xf32>,
```

Same comment for the tests below. Note that you are inserting a single value rather than a vector (`@extract_vector` -> `@insert_vector` -> `@insert_value`?)